_emails/somerville.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Somerville, MA
+permalink: "/somerville"
+name: Letter to Council Members
+city: Somerville
+state: MA
+recipients:
+- cityclerk@somervillema.gov
+- citycouncil@somervillema.gov
+subject: Demanding support for defunding the police in Somerville
+body: |-
+Dear Somerville City Councilors,
```

Comment:
Please put 2 spaces before each line in the body (this will break if merged). Thank you

Reply:
Should be fixed now, sorry I missed that.